package.json

```diff
@@ -1,23 +1,49 @@
 {
   "name": "gatsby-plugin-pinterest",
-  "description": "Load the Pinterest JavaScript to let users Save images to their boards on hover",
-  "version": "1.1.0",
-  "author": "Robin Métral <robin@metral.ch>",
-  "bugs": {
-    "url": "https://github.com/robinmetral/gatsby-plugin-pinterest/issues"
-  },
-  "homepage": "https://github.com/robinmetral/gatsby-plugin-pinterest",
+  "version": "0.0.0-semantically-released",
+  "description": "Gatsby plugin to add Pinterest's embed script.",
+  "main": "./gatsby-browser.js",
   "keywords": [
     "gatsby",
     "gatsby-plugin",
-    "pinterest-widget",
-    "pinterest-save",
-    "gatsby-plugin-pinterest"
+    "embed",
+    "pinterest",
+    "pinterest-widget"
   ],
+  "author": "Robin Métral <robin@metral.ch> (https://robinmetral.com)",
   "license": "MIT",
-  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/robinmetral/gatsby-plugin-pinterest"
+  },
+  "bugs": {
+    "url": "https://github.com/robinmetral/gatsby-plugin-pinterest/issues"
+  },
+  "homepage": "https://github.com/robinmetral/gatsby-plugin-pinterest#readme",
+  "files": [
+    "gatsby-browser.js",
+    "index.js"
+  ],
+  "scripts": {
+    "build": "kcd-scripts build --out-dir .",
+    "contributors:add": "all-contributors add",
+    "contributors:generate": "all-contributors generate",
+    "lint": "kcd-scripts lint",
+    "validate": "kcd-scripts validate"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.8.4"
```

Comment:
We do, since the build script is counting on this dependency.

Right now this is fine to include into `devDependencies`, but if we ever need to do an import, it should be in the `dependencies`, so just as a precaution I've already added it to `dependencies` so we don't forget it.

Reply:
But it should be built before being released, i.e. devDependencies. Users are just going to get the generated `gatsby-browser.js` file. Should we really add Babel to the bundle?

Reply:
It is built before it's released, but if we look at the built code from `gatsby-plugin-instagram-embed`, you'll see this

```js
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

exports.__esModule = true;
exports.onRenderBody = void 0;

var _react = _interopRequireDefault(require("react"));

var onRenderBody = function onRenderBody(_ref) {
  var setHeadComponents = _ref.setHeadComponents,
      setPostBodyComponents = _ref.setPostBodyComponents;
  setHeadComponents([_react.default.createElement("link", {
    key: "gatsby-plugin-instagram-preconnect-0",
    rel: "preconnect",
    href: "https://instagram.com"
  })]);
  setPostBodyComponents([_react.default.createElement("script", {
    key: "gatsby-plugin-instagram",
    dangerouslySetInnerHTML: {
      __html: "\n        window.gatsbyLoadInstagram = function() {\n          var js = document.createElement('script');\n          var firstScript = document.getElementsByTagName('script')[0];\n          js.id = 'gatsby-plugin-instagram';\n          js.src = 'https://instagram.com/embed.js';\n\n          firstScript.parentNode.insertBefore(js, firstScript);\n\n          return true;\n        }\n      "
    }
  })]);
};

exports.onRenderBody = onRenderBody;
```

Like you can see on rule 3, it needs `@babel/runtime`.

Reply:
Isn't this because the plugin imports `react`? In our case we just want to load an external script.

I agree we'll need this is we have imports in the future, but maybe we should start with 0 deps and a smaller bundle size, and keep in mind this might need to be added later on (although in the case of `gatsby-plugin-instagram-embed`, `@babel/runtime` isn't in the dependencies - might be fine since it's already a Gatsby dependency.

Reply:
@robinmetral the bundle size is the same, whether we have it as a dependency or not.

In the case of `gatsby-plugin-instagram-embed`, it's using plain `babel` to build
https://github.com/jlengstorf/gatsby-plugin-instagram-embed/blob/1c53394d1789561a06129efc7244eadf5cf9181f/package.json#L28

https://github.com/jlengstorf/gatsby-plugin-instagram-embed/blob/1c53394d1789561a06129efc7244eadf5cf9181f/.babelrc.js

Reply:
Since `gatsby-plugin-pinterest` is used to build your website, this doesn't mind if we have that dependency or not either.

Reply:
Makes sense. The dependency still looks unnecessary to me, but let's go ahead with it 🙂